modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -194,9 +202,8 @@ const Example = styled('div')(
 
 ### Responsive Styles
 
-We've added a set of
-[responsive utilities](/getting-started/for-developers/resources/responsive-styling/) to facilitate
-container-based and viewport-based responsive styling:
+We've added a set of [responsive utilities](/docs/features-responsive-styling--responsive-container)
```

Comment:
Leaving this note here, but it applies to **all** links in any MDX file which is rendered on the Canvas Site. This link and most (maybe all?) of the links which were changed in MDXes in this PR will need to be reverted back to what they were originally:

```suggestion
We've added a set of [/getting-started/for-developers/resources/responsive-styling/)
```

`/docs/features-responsive-styling--responsive-container` will work when following the link in Storybook, but it won't work on the Canvas Site since `/docs/features-responsive-styling--responsive-container` isn't a valid path on the site.

`/getting-started/for-developers/resources/responsive-styling/` will work for both the site and Storybook since it's a valid path on the site **and** because the mappings in `routes.js` will be used to [rewrite it to the correct path](https://github.com/Workday/canvas-kit/blob/master/.storybook/routes.js#L4) when the MDX is rendered in Storybook (`?path=/docs/features-responsive-styling--responsive-container`).

Reply:
oooo ok!